### DISCUSSION TITLE
add-certificate fails with Dns01Authorization' error when running with v3.6.10 and the Godaddy provider.

### DISCUSSION THREAD

**Discussion by brboyer** [polymind-inc/acmebot#372] [Q&A] [answered]
add-certificate fails with Dns01Authorization' error when running with v3.6.10 and the Godaddy provider. 

Function   WEBSITE_RUN_FROM_PACKAGE setting  
(https://shibayan.blob.core.windows.net/azure-keyvault-letsencrypt/v3/latest.ziplatest.zip)

The Exception 
Orchestrator function 'IssueCertificate' failed: The activity function 'Dns01Authorization' failed: "Exception of type 'System.Net.Http.HttpRequestException' was thrown.".

By deploying version 3.5.7 to functions with same configuration the add-certificate works.

**Comment by shibayan** [comment, marked as answer]
Probably fixed in `v3.6.11`.

**Reply by brboyer** [reply]
Confirming that add-certificate with Godaddy worked with v3.6.11.
Thanks
